prompting/validators/forward.py

```diff
@@ -119,21 +117,34 @@ async def run_step(
         self.device
     )
     for weight_i, reward_fn_i in zip(self.reward_weights, self.reward_functions):
-        reward_i, reward_i_normalized = reward_fn_i.apply(prompt, responses, name)
+        reward_i, reward_i_normalized = reward_fn_i.apply(prompt, responses, task_name)
```

Comment:
Let's remove criteria from reward models. They are not responsible for evaluating criteria and it reduces attack surface

Reply:
```python
prompt_no_criteria = remove_criteria(prompt)
...
reward_i, reward_i_normalized = reward_fn_i.apply(prompt_no_criteria, responses, task_name)
```

Reply:
better still
```python
prompt_no_criteria = task.base_text
```

Reply:
or simply
```
reward_i, reward_i_normalized = reward_fn_i.apply(task.base_text, responses, task_name)
```

Reply:
nice catch! will implement that in a bit